### DISCUSSION TITLE
MAS doesn't work

### DISCUSSION THREAD

**Discussion by IndoDemon7337** [Monika-After-Story/MonikaModDev#9206] [QA - tech support + general] [answered]
So basically, I deleted everything, Just Yuri and the old, corrupted MAS, and I've tried to install the MAS to a non-altered DDLC, but it doesn't go into MAS. When I open DDLC instead of opening with MAS overlay instead it's just the DDLC Monika (It's just when you complete the old DDLC and you're sitting with Monika, no MAS overlay just you and her.) I need help on to what to do, so what should I do.

**Comment by Wingdinggaster656** [comment]
A screenshot of your `/game` directory.
There is a 99% chance that you had a bad install.

**Comment by IndoDemon7337** [comment]
![Help](https://user-images.githubusercontent.com/104285217/170900917-a3c3970b-0f5a-4276-9ce0-129787b6e726.png)
Here is the game directory

**Comment by Wingdinggaster656** [comment, marked as answer]
> ![Help](https://user-images.githubusercontent.com/104285217/170900917-a3c3970b-0f5a-4276-9ce0-129787b6e726.png) Here is the game directory

Completely wrong install.

See this wiki: https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#installation
